apps/web/src/screens/Explore/LayoutCard.tsx

```diff
@@ -0,0 +1,59 @@
+import { FC } from 'react';
+import { Image, Stack, Text } from '@walless/gui';
+
+export interface LayoutCardProps {
+	id: string;
+	name: string;
+	description: string;
+	thumbnail: string;
+	logo: string;
+	loveCount: number;
+	activeUsers: number;
+}
+
+const LayoutCard: FC<LayoutCardProps> = (props) => {
```

Comment:
try destructing `props` object

Reply:
I was thinking about using destructuring as any other components, since this one has lots of attributes (7 in total, maybe more in the future). That's why I think passing an object `props` with be cleaner.
How do you think? Should I change it back to the destructuring syntax?

Reply:
I prefer destructuring